easybuild/tools/filetools.py

```diff
@@ -442,6 +442,30 @@ def search_file(paths, query, short=False, ignore_dirs=None, silent=False, filen
     return var_defs, hits
 
 
+def find_eb_script(script_name):
```

Comment:
Pfft, I don't think that's worth the effort. `find_eb_script` is called once for the bash wrapper script, once for `rpath_args.py` per installation (by `toolchain.prepare()`). Caching isn't going to make a significant difference here (even when multiple installations are being done in one session).